### PR TITLE
grassdata conversion scripts added

### PR DESCRIPTION
**Pull request by neteler**
This PR adds a set of GRASS GIS 8 scripts used to convert the large GRASS GIS 6.4 database that CASAS has built so far to GRASS GIS 8. It converts raster data as well as vector data (topology update as well as attribute transfer from DBF to SQLite).

A `README.md` is included, describing the workflow.

A critical review is welcome.

### CODE REVIEW

**Comment by luisponti** [comment]
Added olive_distribution_pnas.tif with gh CLI.

File: casas_gis_old/scripts/gdata_conversion_G6_to_G8/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+It took approx 40 min on an Intel Core i7 system to upgrade the GRASS
+GIS 6 vector data to GRASS GIS 8 including the rebuilding of vector
+topology and attribute transfer from DBF to SQLite.
+
+Observations:
+
+- a few isolated dbf files remained in a few mapsets which were unconnected
+  to vector maps
+- a few leftover files have been removed (apparently incompletely deleted
```
[the remark this answers is not in the document]
Comment (neteler) [reply casas_gis_old/scripts/gdata_conversion_G6_to_G8/README.md:26]: Okay, so we'll keep an eye on it.

File: casas_gis_old/scripts/gdata_conversion_G6_to_G8/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  vector maps with only helper files left over; documented in the preparation script).
+- In the (few) mapsets which already used SQLite only the topology
+  has been rebuilt:
+  - Here the dbf files remained since no decision could be taken by the
+    contractor if the files are still relevant or not. They are easy to
+    spot by a file extension search.
```
[the remark this answers is not in the document]
Comment (neteler) [reply casas_gis_old/scripts/gdata_conversion_G6_to_G8/README.md:32]: We may look into these special cases together.

File: casas_gis_old/scripts/gdata_conversion_G6_to_G8/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+The upgrade steps were (in a loop):
+
+- set computational region to actual raster map
+- regenerate the map statistics
+
+Observations:
+
+- one raster map is using `r.external` (path: `/Users/luigi/olive_distribution_pnas.tif`) .
+  This map was not yet included in the Zenodo.org database. It could be
+  included and placed in the same mapset.
```
[the remark this answers is not in the document]
Comment (neteler) [reply casas_gis_old/scripts/gdata_conversion_G6_to_G8/README.md:47]: (done in 6c69871b2448d4bcc64eb32fda42daafb1dc6ac7)

**Pull request merged**